airflow/providers/snowflake/hooks/snowflake.py

```diff
@@ -32,6 +32,14 @@
 from airflow.utils.strings import to_boolean
 
 
+def _try_to_boolean(value: Any):
+    if value is None:
+        return False
+    if isinstance(value, (str, type(None))):
```

Comment:
isn't type(None) covered by earlier `is None` check? Can there be an object of NoneType that fails "is None" check?

Reply:
Fixed!